src/utils/index.js

```diff
@@ -78,24 +91,12 @@ exports.flattenData = (data, xAccessor, yAccessor) => {
       var yNode;
       if (typeof y === 'object' && Object.keys(y).length > 0) {
         Object.keys(y).forEach(function (key) {
-          // Check for NaN since d3's Voronoi cannot handle NaN values
-          // Go ahead and Proceed to next iteration since we don't want NaN
-          // in allValues or in xValues or yValues
-          if (isNaN(y[key])) {
-            return;
-          }
           yValues.push(y[key]);
           // if multiple y points are to be plotted for a single x
           // as in the case of candlestick, default to y value of 0
           yNode = 0;
         });
       } else {
-        // Check for NaN since d3's Voronoi cannot handle NaN values
-        // Go ahead and Proceed to next iteration since we don't want NaN
-        // in allValues or in xValues or yValues
-        if (isNaN(y)) {
-          return;
-        }
```

Comment:
is there any reason you remove all the isNan() check. We need this in graph with nested data.